lib/src/metta/runner/builtin_mods/fileio.metta

```diff
@@ -0,0 +1,19 @@
+(@doc file-open
```

Comment:
Functions with side-effects should have `!` exclamation mark at the end of the name. Please add it to `file-open!`, `file-read!` and `file-write!`